test/wave-packerSpec.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable
+ camelcase,
+ new-cap
+ */
+
+/* global
```

Comment:
Instead of globals, you can just use `env`.

In `.eslintrc` add `jasmine: true`.